gcs/src/main/scala/io/monix/connect/gcs/configuration/BlobInfo.scala

```diff
@@ -0,0 +1,148 @@
+package io.monix.connect.gcs.configuration
+
+import java.time.Instant
+
+import com.google.cloud.storage.BlobInfo.CustomerEncryption
+import com.google.cloud.storage.{Acl, BlobId, BlobInfo => GoogleBlobInfo, StorageClass}
+
+import scala.jdk.CollectionConverters._
+
+final case class BlobInfo(name: String,
```

Comment:
I would go with case classes, since there is a risk of causing binary compatiblity issues when they are public to the user and if they evolve, but I would like check that with @alexandru or @Avasil

Reply:
AFAIK it's possible to add a field with default parameter without breaking binary compatibility. If fields are removed then I guess we could ignore them

Do you have any alternative in mind?

Reply:
Not sure about that, that was something I read from: https://github.com/jatcwang/binary-compatibility-guide 
An alternative could be replacing it with a class and a companion object?